### DISCUSSION TITLE
Auto enable "Final Price" toggle

### DISCUSSION THREAD

**Discussion by eliac7** [keybraker/reSkroutzed#10] [Ideas]
Hey, 

Firstly, I would like to express my admiration for the outstanding tool you have developed. I was wondering if there is a way to automatically enable the "Final Price" toggle, even if it is not currently within the scope of the extension's features. This would be incredibly useful as it would allow us to quickly view the final price. 

Thank you for your time

**Comment by keybraker** [comment]
Hey @eliac7, a big thanks for your heartwarming words, I'm really happy to see you are using it.

To answer your question simply, I think it is out of scope for this extension. I truly find that it was somewhat of a double edged sword to go with this name, as it prohibits further functionality to be implemented.

If I had the ability to change the name, and incorporate more features, I probably would, but as it stands I think it is best, not to deviate much from the initial reason of existence.

**Reply by keybraker** [reply]
Buy through skroutz has a tool tip bellow it that displays whether the price from skroutz is the lowest one or not.
